src/darwin/Framework/CHIP/MTRDeviceControllerDataStore.mm

```diff
@@ -0,0 +1,238 @@
+/**
+ *    Copyright (c) 2023 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include "MTRDeviceControllerDataStore.h"
+#import "MTRLogging_Internal.h"
+
+// TODO: FIXME: Figure out whether these are good key strings.
```

Comment:
Can we have a github issue tracking this?

Reply:
Yep, https://github.com/project-chip/connectedhomeip/issues/28973